data/schemas/imports/cantines.json

```
@@ -64,30 +64,30 @@
       "constraints": {
         "required": true
       },
-      "description": "Nombre de repas servis, en moyenne, par jour dans la cantine.",
+      "description": "",
       "example": "300",
       "format": "default",
       "name": "nombre_repas_jour",
-      "title": "Nombre de repas par jour",
+      "title": "Nombre de repas servis, en moyenne, par jour dans la cantine.",
```

les titles du schema `achats.json` n'ont pas de point à la fin :sweat_smile: 
```suggestion
      "title": "Nombre de repas servis, en moyenne, par jour dans la cantine",
```